content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,37 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: 
+              {{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+               `.spec.os.name` should be set to `windows` to indicate that the current Pod uses Windows containers.
+              Please note that there is no effect when setting a value for this field since it is behind a feature gate 
```

Comment:
```suggestion
              Please note that there is no effect when setting a value for this field when the feature gate 
```

Reply:
Thank you, I've committed your suggestions.